### PR TITLE
Bugfix/quantinuum backend example notebook login

### PR DESCRIPTION
**Pull request by yao-cqc**

### CODE REVIEW

File: modules/pytket-quantinuum/examples/Quantinuum Circuit Submissions via pytket.ipynb [hunk 1]
```diff
@@ -10689,7 +10689,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.9.12"
+   "version": "3.9.9"
```
Comment (cqc-melf) [inline modules/pytket-quantinuum/examples/Quantinuum Circuit Submissions via pytket.ipynb:10692]: Why are you downgrading the version?

File: modules/pytket-quantinuum/examples/Quantinuum Circuit Submissions via pytket.ipynb [hunk 2]
```diff
@@ -1237,11 +1241,7 @@
     }
    ],
    "source": [
-    "machine = 'H1-1E'\n",
-    "\n",
-    "backend = QuantinuumBackend(device_name=machine)\n",
-    "\n",
-    "print(machine, \"status:\", backend.device_state(device_name=machine))"
+    "print(machine, \"status:\", QuantinuumBackend.device_state(device_name=machine))"
```
Comment (cqc-melf) [inline modules/pytket-quantinuum/examples/Quantinuum Circuit Submissions via pytket.ipynb:1244]: Have you check that this is changed correctly? Why are you not using the backend you created?

File: modules/pytket-quantinuum/examples/Quantinuum Circuit Submissions via pytket.ipynb (hunk 1 above)
Comment (cqc-melf) [reply modules/pytket-quantinuum/examples/Quantinuum Circuit Submissions via pytket.ipynb:10692]: Okay great!

File: modules/pytket-quantinuum/examples/Quantinuum Circuit Submissions via pytket.ipynb (hunk 2 above)
Comment (cqc-melf) [reply modules/pytket-quantinuum/examples/Quantinuum Circuit Submissions via pytket.ipynb:1244]: Okay, would have been nice to do the status check fist before selecting a device, but I would assume that is not possible with the current API.

**Review by cqc-melf — APPROVED** [review]

File: modules/pytket-quantinuum/examples/Quantinuum Circuit Submissions via pytket.ipynb (hunk 2 above)
Comment (yao-cqc) [reply modules/pytket-quantinuum/examples/Quantinuum Circuit Submissions via pytket.ipynb:1244]: Yes, the current implementation of login has some issues. That will be addressed in a future PR.

**Pull request merged**